core/utils/poly-look/src/css/index.js

```diff
@@ -0,0 +1,10 @@
+import "./styles.css";
+import "./colors.css";
+import "./themes.css";
+import "./buttons.css";
+import "./font.css";
+
+/**
+ * Doesn't really do anything except being a place where css files are referenced so that they're bundled
+ */
+export const cssBundler = () => {};
```

Comment:
This is debatable for sure

Reply:
But what are the alternatives?

Reply:
Wait, you mean the whole file, not the const, right?

Reply:
I mean the const the rest is css imports that are normal, they just need to be somewhere

Reply:
If you do want to keep this (looks like it), you could just export the stuff you're importing here, for good measure. Similarly to how `index.js` files in our libraries generally work.

Reply:
But does it work with css files since there is nothing that gets imported they just get bundled by rollup-css